tests/cql/CqlDateTimeOperatorsTest.xml

```diff
@@ -563,15 +568,15 @@
 	<group name="Uncertainty tests">
 		<test name="DateTimeDurationBetweenUncertainInterval">
 			<expression>days between DateTime(2014, 1, 15) and DateTime(2014, 2)</expression>
-			<output>Interval[ 17, 44 ]</output>
+			<output>Interval[ 16, 44 ]</output>
 			<!-- TODO: How to handle the fact the question is resulting in an
 			undertainty interval and that CQL/ELM seem to provide no direct way of
 			selecting the same value, conceptually an implementation internal;
 			currently Equivalent() results in null from comparing with an Interval. -->
 		</test>
 		<test name="DateTimeDurationBetweenUncertainInterval2">
 			<expression>months between DateTime(2005) and DateTime(2006, 5)</expression>
-			<output>Interval[ 5, 16 ]</output>
+			<output>Interval[ 4, 16 ]</output>
```

Comment:
* lower: (whole) months between @2005-12-31T23:59:59.999 and @2006-05-01T00:00:00.000 = `4`
* upper: (whole) months between @2005-01-01T00:00:00.000 and @2006-05-31T23:59:59.999 = `16`